### PR TITLE
docs(rxRadio): FRMW-190 Migrate rxRadio

### PR DESCRIPTION
**Pull request by ssalinas24**
JIRA: https://jira.rax.io/browse/FRMW-190

* [x] Dev LGTM
* [x] Dev LGTM

### CODE REVIEW

File: src/rxRadio/rxRadio.js [hunk 1]
```diff
@@ -16,11 +16,32 @@ angular.module('encore.ui.rxRadio', [])
...
+ *
+ * It is highly recommended that you use `ng-show` and `ng-hide` for display logic.
+ * Because of the way that `ng-if` and `ng-switch` directives behave with scope, they may
+ * introduce unnecessary complexity in your code.
  *
  * @example
  * <pre>
- * <input rx-radio />
+ *    <input rx-radio id="radDemo" ng-model="radDemo" />
+ *    <label for="radDemo">Label for Demo Radio</label>
```
Comment (bry4n) [inline src/rxRadio/rxRadio.js]: you can remove indentation 

```html
<pre>
<input rx-radio id="radDemo" ng-model="radDemo" />
<label for="radDemo">Label for Demo Radio</label>
</pre>
```

**Comment by ssalinas24** [comment]
@bry4n thank you for the feedback. I have made the changes in my second commit.

**Comment by bry4n** [comment]
@ssalinas24 awesome! :+1: 

LGTM

**Comment by 100stacks** [comment]
LGTM :+1:

**Pull request merged**